packages/web-sdk/authenticator.ts

```diff
@@ -25,7 +36,7 @@ class Authenticator extends EventEmitter {
       typeof window !== "undefined" ? this.loadRefreshTokenFromLocalStorage() : null;
   }
 
-  async authenticate(credentials: Credentials): Promise<{
+  async Login(credentials: LoginCredentials): Promise<{
```

Comment:
Why Login is capital and signUp is not? Can you switch this to lower case?
```suggestion
  async login(credentials: LoginCredentials): Promise<{
```